package/cfg/cis-1.4/master.yaml

```diff
@@ -1501,31 +1501,31 @@ groups:
     scored: false
 
   - id: 1.7.2
-    text: "Do not admit containers wishing to share the host process ID namespace (Not Scored)"
+    text: "Do not admit containers wishing to share the host process ID namespace (Scored)"
     type: "manual"
     remediation: |
      [Manual test]
      Create a PSP as described in the Kubernetes documentation, ensuring that the .spec.hostPID field is omitted or set to false.
     scored: false

```

Comment:
Set `scored: true` for these tests?

Reply:
good catch, I had fixed them for upstream, missed on local. Will fix.